LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Come to think of it...should this use the Creative Commons license since this looks like a docs repo?

Ref:
- https://github.com/kubernetes/website/blob/master/LICENSE
- https://github.com/kubernetes/kubernetes-template-project#kubernetes-template-project